www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
> Maybe put a document on the trade-offs elsewhere and link to it from this document?

good idea, will do

> But - I was deliberately trying to avoid getting into that discussion in this PR.

that didn't quite work out, sorry:)